src/runtime-rs/crates/runtimes/src/shim_mgmt/client.rs

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#![allow(dead_code)]
+
+// Defines the general client functions used by other components acting like
+// clients. To be specific, a client first connect to the socket, then send
+// request to destined URL, and finally handle the request(or not)
+
+use std::{path::Path, path::PathBuf, time::Duration};
+
+use super::server::mgmt_socket_addr;
+use anyhow::{anyhow, Context, Result};
+use hyper::{Body, Client, Method, Request, Response};
+use hyperlocal::{UnixClientExt, UnixConnector, Uri};
+
+/// Shim management client with timeout
+pub struct MgmtClient {
+    /// The socket *file path* on host file system
+    s_path: PathBuf,
+
+    /// The http client connect to the long standing shim mgmt server
+    client: Client<UnixConnector, Body>,
+
+    /// Timeout value for each dial
+    /// The http request will always timeout if the timeout is not generous enough
```

Comment:
What's is the `generous enough` value? This line may make people confused.

Reply:
I changed the comment. Suggesting 200ms will do for most requests.